dev/com.ibm.ws.jsp/bnd.bnd

```diff
@@ -99,9 +99,9 @@ Import-Package: \
    org.apache.el.lang;version="2.2", \
    org.apache.el.parser;version="2.2", \
    org.apache.el.util;version="2.2", \
-   org.apache.jasper;version="[2.2,3.2)", \
-   org.apache.jasper.el;version="[2.2,3.2)", \
-   org.apache.jasper.runtime;version="[2.2,3.2)", \
+   org.apache.jasper;version="2.2", \
+   org.apache.jasper.el;version="2.2", \
+   org.apache.jasper.runtime;version="2.2", \
```

Comment:
Was this just missed when we cleaned up the code to be shared?

Reply:
Rather than update the imports every time a new pages factory is created, I opted to just set a minimum here. 

If I think this as `[2.2,3.2)` then the exported 4.0 bundles would not resolve here.